src/lib/model/InboundThirdpartyTransactionModel.js

```diff
@@ -82,13 +82,18 @@ class InboundThirdpartyTransactionModel {
         let mojaloopErrorCode = Errors.MojaloopApiErrorCodes.INTERNAL_SERVER_ERROR;
         if (err instanceof HTTPResponseError) {
             const e = err.getData();
-            if (e.res && e.res.body) {
-                try {
-                    const bodyObj = JSON.parse(e.res.body);
-                    mojaloopErrorCode = Errors.MojaloopApiErrorCodeFromCode(`${bodyObj.statusCode}`);
-                }
-                catch (ex) {
-                    this._logger.push({ ex }).log('Error parsing error message body as JSON');
+            if(e.res && (e.res.body || e.res.data)) {
```

Comment:
Does this mean that there's some mismatch some where in the construction of responses? I would of thought that there's some standard. 

Anyway I won't dig too deep into that. I'm fine with this.

Reply:
@kleyow thanx!